compiler/lsp-tests/src/Main.hs

```diff
@@ -708,8 +708,10 @@ multiPackageTests damlc = testGroup "multi-package"
               Just escapedFpB <- pure $ escapeURIString isUnescapedInURIComponent <$> uriToFilePath (docB ^. uri)
               -- code lenses are a good test since they force LF compilation
               r <- getCodeLenses docB
+              -- TODO: test broken by --hide-all-packages being turned on. Why??
```

Comment:
I don't understand this!

Reply:
What I mean is. By turning on `--hide-all-packages`, this test broke...
So I made a temp hack to suppress the failure, showing what the new behaviour is.
But I have no idea if this new behaviour is correct. Or more generally what this test is about!

Reply:
This test checks that the IDE can load multiple packages at once. In this case the IDE will not be started in the project directory but in a parent directory, e.g., you open vscode in `foobar` and you have two projects at `foobar/a` and `foobar/b`. This means that `init` will do nothing since it is run in `foobar` and not in `foobar/a`. The proper fix for this is to change where we do `init` but I deliberately left this out of #4445 since it would have conflicted even more with this PR.

For now, I think the best fix is to add a `--package` flag to the `build-options` field of `b` and mention this in the docs at https://github.com/digital-asset/daml/blob/master/docs/source/daml/daml-studio.rst#working-with-multiple-packages. We can then shuffle around pkg db initialization afterwards to make this unnecessary.

Reply:
https://github.com/digital-asset/daml/pull/4481 adds another test for starting the IDE in the respective project directory. In that case `init` should work correctly and you should not need to add a `--package` flag. Probably a good idea to merge #4481 first and rebase this on top of it to make sure this is the case.

Reply:
These tests all work now.